tools/internal_ci/helper_scripts/prepare_build_windows.bat

```diff
@@ -34,6 +34,12 @@ netsh interface ip add dnsservers "Local Area Connection 8" 8.8.4.4 index=3
 @rem Needed for big_query_utils
 python -m pip install google-api-python-client
 
+@rem Install Python 3.7
+chocolatey install -y python3
```

Comment:
Is there a way to pin this to 3.7 specifically so that it won't automatically get bumped if an update gets shipped?

Reply:
On second thought, it would be immaterial if we are hoping Kokoro to preinstall it for us sooner than 3.8 comes along.